File: cmd/analyze/code-samples/flagship/ios/SDK_V2/sample.m
/// init config object and set the running mode and timeout
FSConfig * config = [[FSConfig alloc] init:FlagshipModeDECISION_API timeout:0.2];

/// Start the sdk
[[Flagship sharedInstance] startWithEnvId:@"you envId" apiKey:@"your apiKey" visitorId:@"visitorId" config:config onStartDone:^(enum FlagshipResult result) {

  if (result == FlagshipResultReady){

    dispatch_async(dispatch_get_main_queue(), ^{

              /// update UI
    });
  }else{

            /// An error occurs or the SDK is disabled
  }

}];

// Retrieve modification and activate
 NSString * title = [[Flagship sharedInstance] getModification:@"btnColor" defaultString:@"More Infos" activate:YES];
 NSString * title = [[Flagship sharedInstance] getModification:@"showBtn" defaultBool:TRUE activate:YES];
 NSString * title = [[Flagship sharedInstance] getModification:@"btnSize" defaultInt:13 activate:YES];
 NSString * title = [[Flagship sharedInstance] getModification:@"btnSizeFloat" defaultFloat:14.5 activate:YES];




// END

File: example/src/ios/SDK_V2/sample.swift
/// Update the context when basket value change
Flagship.sharedInstance.synchronizeModifications { (result) in

  if result == .Updated{

    // Update the UI for users that have basket over or equal 100
    if (Flagship.sharedInstance.getModification("freeDelivery", defaultBool: false, activate: true)){

      DispatchQueue.main.async {

        /// Show your message for free delivery

      }
    }
  }
}


// Retrieve modification and activate
let title = FlagShip.sharedInstance.getModification("btnColor", defaultString: "#000000", activate: true)

// Retrieve modification and activate
let title = FlagShip.sharedInstance.getModification("btnSize", defaultDouble: 13, activate: true)
let title = FlagShip.sharedInstance.getModification("btnSizeFloat", defaultFloat: 13.1, activate: true)
let title = FlagShip.sharedInstance.getModification("btnSizeInt", defaultInt: 13.00, activate: true)




// END
